recipes/pytest-watcher/meta.yaml

```diff
@@ -15,13 +15,13 @@ build:
   entry_points:
     - ptw = pytest_watcher:run
     - pytest-watcher = pytest_watcher:run
-  script: {{ PYTHON }} -m build --wheel --no-isolation . && {{ PYTHON }} -m pip install dist/pytest_watcher-{{ version }}-py3-none-any.whl -vv
```

Comment:
We don't need all that for the conda package.

Reply:
Thanks! I didn't think the simple `pip install .` would work, since the project specifies a build dependency on `poetry`. But if it works, it's certainly a lot simpler!

Reply:
Try generating the recipes with `grayskull`. It should add all that is necessary and nothing more. And yeah, poetry is still needed b/c that is the build backend, pip will invoke it for us.

Reply:
OK, awesome. I started this recipe with `grayskull` but the initial builds failed locally. Must have been something wrong with my configuration. Thanks for merging!

Reply:
Probably the lack of poetry if that was declared in a pyproject.toml file. This is a known 'bug' and @marcelotrevisani is working on it.